web-console/4.0-designs/overview/overview.md

```diff
@@ -0,0 +1,87 @@
+  * If no resources have labels, items will be shown in a regular list view listed alphabetically by resource type and then name and the `Group by label` dropdown will be hidden.
+* On hover, the row will be highlighted (#EDF8FF) and the icon fa-angle-right (pf-blue-400, #0088CE) will appear at the end of the row. When a row is selected (#DEF3FF), the side panel will open with a summary of that resource's overview, pushing the content over to the left.
+* If a user is not in the context of a selected project, the user will be shown an [empty state](http://openshift.github.io/openshift-origin-design/web-console/4.0-designs/empty-states/empty-states) until they select a project from the project dropdown.
+![empty state](img/ES_overview.png)
+
+### List columns
+
+* The first column will include resource badges and names. Users can redirect to a resource's details page by clicking the resource name.
+* The second column will display status updates and other relevant notifications and events such as crash looping containers and deployment failures. Users can hover on a status or notification to see more information.
+* The third column will be a summary of the item's metrics, including memory, CPU, and network. If metrics are not enabled, this column will be blank. If metrics are enabled but are not currently available, users will see "--" in place of the values.
+* The fourth column will show how many pods have deployed for that item. Users can click on the pods link to be directed to the resource's pods page.
+
+### Route URL
+
+![route](img/route.png)
+* Users may navigate to the application by clicking the route URL link (pf-blue-400, #0088CE) across from the application name. The icon fa-external-link (pf-blue-400, #0088CE) will be to the right of the URL to indicate that the link redirects users to an external location and opens a new browser tab.
+* Users may edit which route URL is displayed by clicking the black (pf-black-900, #292E34) pficon-edit icon. A popover will open where users may set their default route from a list of their routes. Users may also copy route URLs from this popover. Users may click Cancel or ‘x’ or click anywhere else on the screen to close the popover, or click Save to confirm their selection.
+  * The default route will be set only for the current user.
+
+
+
+## Side panel view
+
+### General
+
+* When a row is clicked, the side panel will open with a summary for that resource.
+* The object badge and name are displayed at the top of the panel, along with an actions dropdown for users to take action on the object.
+* The side panel has two main tabs: `Overview` and `Resources`.
+* The `Overview` tab is the default view, and the list view content is pushed to the left.
+* The selected row remains highlighted, and the metrics column is hidden.
+* When necessary, the resource name, notifications, and route URL will wrap.
+* The `Group by label` dropdown and filter bar are not accessible when the side panel is open.
```

Comment:
It worries me that you can have a filter applied and not see it. I personally would rather keep these controls visible.

Reply:
@spadgett Would you be suggesting to keep them visible but disabled? Or keep them visible and enabled? 

I think the fear with keeping them visible and enabled is basically users think they can filter on things inside the side panel or they could filter out the action item that is selected which wouldn't make sense.

Reply:
@spadgett I agree it's a potential concern.  We could start with this, and then test for feedback?

Reply:
I'd keep them visible and enabled.

I think the confusion of having something filtered out and not realizing it is worse than possibly thinking you can filter properties in the details. Since the input is with the list rows and not in the details pane, I'm not sure the latter is really an issue. We also don't let you filter individual object properties anywhere else in the UI. It's always lists.

Today we keep the selected item visible always regardless of filter.

Reply:
@spadgett Thanks that makes sense, all fair points. I'd be fine with going either way to start and then testing regardless to confirm :+1:

Reply:
Agreed, all good points. I'm leaning towards leaving them in and seeing if testing says otherwise. I'm not sure that when we made this decision initially we considered all the various factors that Sam brought up. I'll update this if everyone agrees with this approach @beanh66 @serenamarie125 @spadgett

Reply:
Oh no! Didn't see Sam's comment above before I commented. I'm good either way!